extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -107,7 +109,17 @@ public static PhotoModel convertToPhotoModel(
         albumId.orElse(null),
         false  /*inTempStore*/,
         null  /*sha1*/,
-        mediaItem.getUploadedTime());
+        getUploadTime(mediaItem));
+  }
+
+  private static Date getUploadTime(GoogleMediaItem mediaItem)
+  {
+    Date uploadTime = null;
+    try {
+      uploadTime = FORMAT.parse(mediaItem.getMediaMetadata().getCreationTime());
+    }catch (Exception e) {
```

Comment:
the empty catch is concerning a bit worrying. wdyt of documenting what this catch would mean (eg: the gphotos servers had a breaking API change: the string format changed?)

Reply:
Added a comment specifying the behavior and expectation as below - 
this should never be hit since creationTime field should never be null. If we reach here, take a look into the string format change for the DateFormatter

Reply:
Changed the code according to our offline conversation